### PR TITLE
Set the hotkey for saving a file to allow for either Ctrl+S or Cmd+S

### PR DESCRIPTION
**Pull request by RobertBlackhart**

### CODE REVIEW

File: src/ViperIDE.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
         <div>
             <button onclick="app.toggleSideMenu()"><i class="fa-solid fa-bars"></i></button>
             <span class="logo">ViperIDE</span>
-            <button title="Save File [Ctrl+S]"   onclick="app.saveCurrentFile()" id="btn-save"><i class="fa-solid fa-download"></i></button>
+            <button title="Save File [Ctrl/Cmd+S]"   onclick="app.saveCurrentFile()" id="btn-save"><i class="fa-solid fa-download"></i></button>
```
Comment (vshymanskyy) [inline src/ViperIDE.html]: Please note that this is overridden during internationalization:

https://github.com/vshymanskyy/ViperIDE/blob/cf6909562db197498e59f430550e6ef4ecb7f36f/src/app.js#L818

It also presents a chance to detect the OS dynamically and replace with `Ctrl` or `Cmd` accordingly

**Review by vshymanskyy — CHANGES_REQUESTED** [review]

File: src/ViperIDE.html (hunk 1 above)
Comment (RobertBlackhart) [reply src/ViperIDE.html]: Ah, OK. I was wondering how that worked. This makes sense. I'll make some changes here including the dynamic detection.

**Comment by RobertBlackhart** [comment]
The new changes have it look like this on Mac:
image
and this on non-Mac:
image

**Review by vshymanskyy — APPROVED** [review]

**Comment by vshymanskyy** [comment]
Thank you for your contributions!

**Pull request merged**